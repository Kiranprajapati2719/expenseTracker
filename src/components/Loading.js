import React from 'react';

export default () => (
  <div className="parent">
  <div className="hollowLoader">
    <div className="largeBox"></div>
    <div className="smallBox"></div>
  </div>
  </div>
);