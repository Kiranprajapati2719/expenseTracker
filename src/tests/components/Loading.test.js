import Loading from '../../components/Loading';
import React from 'react';
import {shallow} from 'enzyme'; 

test('should render header correctly', () => {
  const wrapper = shallow(<Loading />);
  expect(wrapper).toMatchSnapshot();
});