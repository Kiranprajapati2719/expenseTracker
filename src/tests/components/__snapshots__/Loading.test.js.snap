// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div
  className="parent"
>
  <div
    className="hollowLoader"
  >
    <div
      className="largeBox"
    />
    <div
      className="smallBox"
    />
  </div>
</div>
`;
